lib/icons/tag.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function TagIcon() {
+  return (
+    <svg
+      className="icon-tag"
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+    >
+      <rect x="0" fill="none" width="24" height="24" />
+      <path d="M18.29,5.71v5.17l-7,7L6.12,12.71l7-7h5.17m2-2h-8L4,12a1,1,0,0,0,0,1.41L10.59,20A1,1,0,0,0,12,20l8.29-8.29v-8Zm-4.5,5.5a1,1,0,1,0-1-1A1,1,0,0,0,15.79,9.21Z" />
+    </svg>
+  );
+}
```

Comment:
Fun fact! Even though this code is 100% fine it's also possible to rid ourselves of some of the extra syntax. This is a style thing.

```js
export const TagIcon = () => (
	<svg
		…
	</svg>
);
```

Reply:
Gonna leave this as it is since it's the way all the other icons are doing it.